packages/storage/src/chrome/types.ts

```diff
@@ -15,9 +17,10 @@ export interface OriginRecord {
 
 export interface LocalStorageState {
   wallets: WalletJson[];
-  grpcEndpoint?: string;
+  grpcEndpoint: string | undefined;
   frontendUrl: string;
-  passwordKeyPrint?: KeyPrintJson;
-  fullSyncHeight?: number;
+  passwordKeyPrint: KeyPrintJson | undefined;
+  fullSyncHeight: number | undefined;
   knownSites: OriginRecord[];
+  params: Jsonified<AppParameters> | undefined;
 }
```

Comment:
in order to successfully return undefined from the storage getter, undefined had to be explicitly added to the state type.